src/cmd/cmd.cpp

```diff
@@ -155,6 +157,8 @@ void help()
     std::cout << "  --password, -p [pass]  Use [pass] as password" << std::endl;
     std::cout << "  -n                     Use netrc (5) for login" << std::endl;
     std::cout << "  --non-interactive      Do not block execution with interaction" << std::endl;
+    std::cout << "  --nonshib, -ns         Use Non Shibboleth WebDAV authentication" << std::endl;
```

Comment:
Maybe not strictly necessary, but adding the nonshib switch (which is also theme customizable) is a lot easier and understandable from a user perspective than remembering and adding '--davpath remote.php/nonshib-webdav'. I don't need the davpath option, I added it while I was at it to do you a favor, I would regret missing out on the nonshib switch.

On Nov 2, 2015 23:10, Thomas Müller <notifications@github.com> wrote:

In src/cmd/cmd.cpp<https://github.com/owncloud/client/pull/4060#discussion_r43690443>:

> @@ -155,6 +157,8 @@ void help()
>      std::cout << "  --password, -p [pass]  Use [pass] as password" << std::endl;
>      std::cout << "  -n                     Use netrc (5) for login" << std::endl;
>      std::cout << "  --non-interactive      Do not block execution with interaction" << std::endl;
> +    std::cout << "  --nonshib, -ns         Use Non Shibboleth WebDAV authentication" << std::endl;


As soon as the dav path is customizable there is no need for the nonshib switch from my pov

—
Reply to this email directly or view it on GitHub<https://github.com/owncloud/client/pull/4060/files#r43690443>.